tapalcatl_server/handler.go

```diff
@@ -116,6 +121,10 @@ func logBool(x bool) string {
 	}
 }
 
+func logDuration(x time.Duration) string {
+	return fmt.Sprintf("%d", x.Nanoseconds())
```

Comment:
Yea, I think seeing zero ms is fine... And actually the more I think about it the more helpful that would be because we'd also get a "free" count metric for that operation even if the timing says zero milliseconds.